codingtest_ajw/1일차/찍수-홀수.js

```diff
@@ -0,0 +1,11 @@
+function solution(num) {
+    
+    var answer = '';
+    if(num%2==0)
+        answer="Even"
+    else
+        answer="Odd"
+    return answer;
```

Comment:
```suggestion
    let answer = '';
    
    if (num%2==0) {
        answer="Even"
    } else {
        answer="Odd"
    }
    
    return answer;
```
지금 코드처럼 개행이 있는 경우엔 { } 로 감싸주시면 될 것 같습니다. 혹은
```suggestion
    let answer = '';
    
    if (num%2==0) answer="Even"
    else answer="Odd"
    
    return answer;
```
로 작성 해주셔도 될 것 같습니다.